index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-uWxY/CJNBR+1zjPWmfnSnVxwRheevXITnMqoEIeG1LJrdI0GlVs/9cVSyPYXdcSF" crossorigin="anonymous">
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="style.css">
+    <title>Page Titles</title>
+    <script async src="script.js"></script>
+    <noscript>You need to enable JavaScript to view the full site.</noscript>
+</head>
+
+<body class="body">
+    <img class="topimage" src="https://i.ibb.co/Wzb3T0J/portfolio-top-image2.png" alt="portfolio-top-image2">
+    <div class="w-60 title">
+        <h1><em>Personal Statement</em></h1>
+        <p>I am currently a SMT Operator at ESI Electronics in Waterbury Connecticut. My current goal is to graduate
+            from
+            the University of Vermont's Coding Bootcamp <br>
+            and get a job as a web developer.</p>
+    </div> <br>
+    <div class="electronic-manufactoring">
+        <div class="w-50 work-experience">
+            <h2><u>Work Experience</u></h2>
```

Comment:
Add sections for Projects and maybe a skills section that list skills you've learned in the web dev course